sdk/identity/azure-identity/pom.xml

```diff
@@ -92,6 +92,7 @@
       <groupId>org.linguafranca.pwdb</groupId>
       <artifactId>KeePassJava2</artifactId>
       <version>2.1.4</version> <!-- {x-version-update;org.linguafranca.pwdb:KeePassJava2;external_dependency} -->
+      <optional>true</optional>
```

Comment:
Hi, @g2vinay, the latest released KeePassJava2 is in 2018, could you please implement the function without KeePassJava2?

Refs: https://github.com/jorabin/KeePassJava2/releases